website/static/js/fangorn.js

```diff
@@ -2228,6 +2253,14 @@ tbOptions = {
     uploads : true,         // Turns dropzone on/off.
     columnTitles : _fangornColumnTitles,
     resolveRows : _fangornResolveRows,
+    // Sets location under root to find treebeard data from API. For WB v1 "attributes"
+    dataLocation : function(value) {
+        if ('attributes' in value) {
+            return value.attributes;
+        } else {
```

Comment:
@felliott So the value sent to a function will be returned if the function doesn't explicitly return something?